### PR TITLE
deselect selected item in recycler view when switching tabs

### LINKED ISSUE #31: Cancel action bars / remove selections on home screen tabs when switching between them

### PR DESCRIPTION
**Pull request by NelzkieLabs**

### CODE REVIEW

File: WhatsApp/app/src/main/java/com/example/whatsapp/ui/fragments/home/calls/CallsFragment.kt [hunk 1]
```diff
@@ -106,6 +106,11 @@ class CallsFragment : BaseFragment<FragmentCallsBinding>(), ActionMode.Callback,
 
     }
 
+    override fun onPause() {
+        tracker?.clearSelection()
+        super.onPause()
```
Comment (Josh-Owen) [inline WhatsApp/app/src/main/java/com/example/whatsapp/ui/fragments/home/calls/CallsFragment.kt]: I don't think this way will work as `onPause` is also called when you move the app into the background and we would not want to clear our selection in that instance

File: WhatsApp/app/src/main/java/com/example/whatsapp/ui/fragments/home/chats/ChatsFragment.kt [hunk 2]
```diff
@@ -86,6 +86,11 @@ class ChatsFragment : BaseFragment<FragmentChatsBinding>(), ActionMode.Callback
         //endregion
     }
 
+    override fun onPause() {
+        tracker?.clearSelection()
```
Comment (Josh-Owen) [inline WhatsApp/app/src/main/java/com/example/whatsapp/ui/fragments/home/chats/ChatsFragment.kt]: Same situation here as in CallsFragment

**Review by Josh-Owen — CHANGES_REQUESTED** [review]

File: WhatsApp/app/src/main/java/com/example/whatsapp/ui/fragments/home/calls/CallsFragment.kt (hunk 1 above)
Comment (NelzkieLabs) [reply WhatsApp/app/src/main/java/com/example/whatsapp/ui/fragments/home/calls/CallsFragment.kt]: This should fix it c34973a16b8752e1364bbf7904a2549e90ab874b

File: WhatsApp/app/src/main/java/com/example/whatsapp/ui/fragments/home/chats/ChatsFragment.kt (hunk 2 above)
Comment (NelzkieLabs) [reply WhatsApp/app/src/main/java/com/example/whatsapp/ui/fragments/home/chats/ChatsFragment.kt]: Update c34973a16b8752e1364bbf7904a2549e90ab874b

**Review by OskarLasota — APPROVED** [review]
looks good

**Review by Josh-Owen — APPROVED** [review]

**Pull request merged**